ovn-org/ovn-kubernetes
# Metric: Add Egress IP count metric

Egress IP count metric represents the total Egress IPs seen in the cluster. This possibly includes multiple Egress IPs per EgressIP CR.

This metric will allow us to understand is egress IP being used.
It will also let us understand the amount of egress IPs that are active in the cluster.

This metric is not captured by kube-state-metrics.

Signed-off-by: Martin Kennelly <mkennell@redhat.com>


**- How to verify it**
1) Exec into ovnkube-master pod which is the active leader. 
2) Curl the metrics endpoint and grep for 'num_egress_ips'.
3) It should have a value of 0.
4) Apply an EgressIP CR with X amount of egressIPs.
5) Repeat step 2 and expect a value of X.


**- Description for the changelog**
None
